docs/modules/ROOT/pages/performance_analysis/dpmAcaGrpcTest.adoc

```diff
@@ -0,0 +1,215 @@
+= ALCOR CONTROL AGENT-ALCOR DATAPLANE MANAGER Test Report 
+Xiaodong Zhang <xzhang2@futurewei.com>
+v0.1, 2020-09-15
+:toc: right
+:imagesdir: ../../images
+
+*ALCOR CONTROL AGENT-ALCOR DATAPLANE MANAGER Test Report*
+
+**Abstract：**This document contains Alcor Dataplane Manager-Alcor Control Agent grpc performance test result and analysis
+
+[arabic]
+. *Environment description*
+
+[cols=",",options="header",]
+|===
+|*IP address* |
+|*A* |*10.213.43.162*
+|*B* |*10.213.43.163*
+|*C* |*10.213.43.164*
+|*D* |*10.213.43.166*
+|*E* |*10.213.43.187*
+|F |*10.213.43.188*
+|===
+
+[cols=",,",options="header",]
+|===
+|*Hardware Configuration:* | |
+| |*A, B, C, D, E running ALCOR CONTROL AGENT* |*F running as ALCOR DATAPLANE MANAGER*
+|*CPU* |40 cores |56 cores
+|*Model Name* |Intel(R) Xeon(R) CPU E5-2690 v2 @ 3.00GHz |Intel(R) Xeon(R) CPU E5-2697 v3 @ 2.60GHz
+|*cpu MHz* |2231.772 |2599.079
+|*Memory* |192GB |386GB
+|*Network* |NetXtreme BCM5719 Gigabit Ethernet PCIe (GB network) |82599ES 10-Gigabit SFI/SFP+ Network Connection
+|*Storage* |LSI raid (no ssd) |AVAGO (no ssd)
+|*Software Configuration:* | |
+|*System* |Ubuntu18 LTS |
+|*Server* |ALCOR CONTROL AGENT on A, B, C, D, E |
+|*Client* |ALCOR DATAPLANE MANAGER on F |
+|===
+
+[arabic, start=2]
+. *Test step:*
+
+F send goal state message to A-E at the same time concurrently after first warming up then wait for the response, goal state message is different in each payload
+
+On A-E there are 2600 ACA running on each box, ACA code has been revised to cut off the ovsdb and mq operations
+
+image::p1.png["Test Deployment",width=488,height=302]
+
+[arabic, start=3]
+. *Test results*
+
+[cols=",,,,,,,,",options="header",]
+|===
+|*Log Time Cost Summary (ms)* | | | | | | | |
+|*Thread* |*1 -1^st^* |*1-2^nd^* |*32 -1^st^* |*32-2^nd^* |*56 -1^st^* |*56-2^nd^* |*128 -1^st^* |*128-2^nd^*
+|*Average* |11.14 |10.37 |16.50 |14.40 |26.54 |25.60 |70.64 |61.54
+|*MIN* |9 |9 |10 |9 |10 |9 |10 |9
+|*MAX* |48 |18 |223 |116 |364 |160 |603 |352
+|*MEDIAN* |11 |10 |11 |11 |11 |11 |11 |11
+|*>AVERAGE* |28.88% |40.35% |21.48% |20.09% |21.80% |20.09% |21.03% |20.00%
+|*<AVERAGE* |71.12% |59.65% |78.52% |79.91% |78.20% |79.91% |78.97% |80.00%
+|*99% TILE* |13 |12 |67 |43 |107.04 |116.03 |331 |312
+|*95% TILE* |12 |12 |36 |31 |83 |89 |305 |277
+|*90% TILE* |12 |11 |32 |28 |78 |84 |292 |262
+|===  
+
+
+[arabic, start=4]
+. *Test results analysis*
+[loweralpha]
+.. {blank}
++
+____
```

Comment:
what is "____" for? I think this creates some formatting issue if you check https://github.com/haboy52581/alcor/blob/feature/add-adoc/docs/modules/ROOT/pages/performance_analysis/dpmAcaGrpcTest.adoc#L4